src/server/ocpp/utils/OCPPUtils.ts

```diff
@@ -692,6 +692,24 @@ export default class OCPPUtils {
 
   public static updateTransactionWithStopTransaction(transaction: Transaction, stopTransaction: OCPPStopTransactionRequestExtended,
       user: User, alternateUser: User, tagId: string): void {
+    // Handle Signed Data
+    for(const meterValue of (stopTransaction.transactionData as OCPPMeterValue[])){
+      if(meterValue.sampledValue){
```

Comment:
You development setup does not have the mandatory vscode extensions installed and allowed: that PR is very badly formatted.

Reply:
reformated